src/components/views/analytics/AnalyticsView.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useMemo, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { Alert, Box, Grid, Skeleton } from '@mui/material';
+
+import { hooks } from '../../../config/queryClient';
+import { CONTAINER_HEIGHT } from '../../../constants/style';
+import {
+  getMembersWithMostViews,
+  groupActionByTimeInterval,
+  intervals,
+  topMembersRangeOptions,
+} from '../../../utils/chart';
+import MostFrequentUsersChart from './MostFrequentUsersChart';
+import SelectDisplayedUsersLimit from './SelectDisplayedUsersLimit';
+import SelectInterval from './SelectInterval';
+import ViewsOverTimeChart from './ViewsOverTimeChart';
+
+const AnalyticsView = (): JSX.Element => {
+  const [interval, setInterval] = useState(intervals[0]);
+  const [displayedUsersLimit, setDisplayedUsersLimit] = useState(
+    topMembersRangeOptions[0]
+  );
+
+  const { t } = useTranslation();
+  const { data, isLoading } = hooks.useAppActions();
+
+  const actionsGroupedByInterval = useMemo(
+    () => groupActionByTimeInterval(data || [], interval.groupBy),
+    [data, interval.groupBy]
+  );
+
+  const topFrequentUsers = useMemo(
+    () =>
+      getMembersWithMostViews(data || [], Number(displayedUsersLimit.value)),
+    [data, displayedUsersLimit.value]
+  );
+
+    if (data) {
+    return (
+      <Grid container>
+        <Grid item xs={12} sm={12} md={6} lg={6} xl={6}>
```

Comment:
I think it is enough to specify the columns up to the one you want to modify in this case, `xs={12} md={6}` would be enough, as sm would take `12` and lg, xl would take `6`. Check and let me know.